modules/core/pom.xml

```diff
@@ -189,18 +189,18 @@
 							org.apache.synapse.carbonext.*,
                             org.apache.synapse.continuation.*,
                             org.apache.synapse.debug.*,
-                            org.apache.synapse.messageflowtracer.*
+                            org.apache.synapse.messageflowtracer.*,
                         </Export-Package>
-						<Import-Package>
-							!javax.xml.namespace,
-							javax.xml.namespace;
-							version=0.0.0,
-							!javax.xml.soap,
-							javax.xml.soap; version=${javax.xml.soap.version},
-							!org.apache.commons.io,
-							org.apache.commons.io; version=0.0.0,
-							org.quartz.*;version="${quartz.version}",
-							*;resolution:=optional,
+                        <Import-Package>
+                            !javax.xml.namespace,
+                            javax.xml.namespace;
+                            version=0.0.0,
+                            !javax.xml.soap,
+                            javax.xml.soap; version=${javax.xml.soap.version},
+                            !org.apache.commons.io,
```

Comment:
Let's order imports first list imports that restrict (!javax.xml.namespace, javax.xml.soap etc.)

Reply:
Done in https://github.com/wso2/wso2-synapse/pull/1466/commits/a79e6b69b66664765e7c3a3c9667225f44299363